src/components/dashboard/index.js

```diff
@@ -139,6 +143,45 @@ const Dashboard = (props) => {
     const [detailsApp, setDetailsApp] = useState(null);
     const [detailsAnalysis, setDetailsAnalysis] = useState(null);
     const [pendingAnalysis, setPendingAnalysis] = useState(null);
+    const [terminateAnalysis, setTerminateAnalysis] = useState(null);
+
+    const [terminateAnalysisDlgOpen, setTerminateAnalysisDlgOpen] =
+        React.useState(false);
+    const { mutate: analysesCancelMutation, isLoading: analysisLoading } =
+        useMutation(cancelAnalysis, {
+            onSuccess: (analyses, { job_status }) => {
```

Comment:
Minor, but looks like the `analyses` and `job_status` variables aren't used anywhere.  Not sure if a success announcer was meant to be here or if we can remove these.